ios/brave-ios/Sources/BraveWallet/Extensions/KeyringServiceExtensions.swift

```diff
@@ -19,4 +20,14 @@ extension BraveWalletKeyringService {
     let allAccountsForKeyring = await allAccounts().accounts.filter { $0.keyringId == keyringId }
     return !allAccountsForKeyring.isEmpty
   }
+
+  /// Return a list of all accounts with checking if Bitcoin testnet is enabled
+  /// The list of account will not include Bitcoin Testnet Accounts if Bitcoin testnet is disabled.
+  func allAccounts(checkBTCTestnet: Bool) async -> [BraveWallet.AccountInfo] {
```

Comment:
Should this just default to `true`? We don't provide false anywhere. It may be used in unit tests later, but I think when Bitcoin Testnet network is disabled we likely don't need the account(s) at all either 🤔

Reply:
was trying to distinguish this method from `BraveWalletKeyringService.allAccounts()` which returns a `BraveWalletAllAccountsInfo`

Reply:
as discussed change the signature to `accountInfos()` [8193cb9](https://github.com/brave/brave-core/pull/22971/commits/8193cb9c02fb3fbabeac81084bb564414ca2c7e6)